npm/qsharp/src/language-service/language-service.ts

```diff
@@ -274,6 +286,18 @@ export class QSharpLanguageService implements ILanguageService {
       log.error("Error in onDiagnostics", e);
     }
   }
+
+  async onTestCallables(callables: ITestDescriptor[]) {
+    try {
+      const event = new Event("testCallables") as LanguageServiceEvent & Event;
```

Comment:
Should this and onDiagnostics (line 278 above) now be the specific event type? (e.g. LanguageServiceDiagnosticEvent above and the TestCallables version here)